docs/projects/even-odd/_posts/2018-12-31-python.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Even-Odd in Python
+layout: default
+last-modified: 2018-12-31
+tags: ['python', 'even-odd', 'syntax']
+authors: "Chase Stevens"
+---
+
+# Even_Odd
+
+Oftentimes when dealing with things and numbers, we want to split up what we're working with into two groups. This property, known as parity, is useful to know. With Python, we can create a function to check the parity of any given integer.
+
+    def even_odd(x):
+        return "Even" if x % 2 == 0 else "Odd"
+
+Here we have a function named 'even_odd' that takes the parameter "x". The function checks to see if x % 2 is equal to 0. If this is true, we return the value "Even", and if it is false, then we return "Odd."
+
+### Modulo
+One symbol which sticks outs is "%". In Python, and in many other programming languages, the symbol % represents a "modulo" operation, which returns the remainder of the first number divided by the second number.
+
+Let's say we wanted to find out the remainder of 5 divided by 2. We'd calculate that 2 goes into 5 twice with a remainder of one. We can express this with the modulo operator as 5 % 2 = 1. Similarly, we could check the modulo of 9 and 3. Since 3 goes into 9 evenly, we would get an answer of 0, expressed as 9 % 3 = 0.
```

Comment:
> 2 goes into 5 twice with a remainder of one
For consistency's sake, I think one should be 1. That way it matches 2 and 5.

Alternatively they could all be written out, but I prefer the above.

Reply:
Thanks, fixed

Reply:
Looks great!